### ISSUE TITLE
README note about bodyParser() outdated.

### ISSUE THREAD

**Issue by MichaelJCole** [pouchdb/express-pouchdb#120]
Hi, Express 4 seems to have outdated the README note about bodyParser().

I got this code to work (at least for the trivial test of /db).  Look right?  

Is it possible to run tests against the server to see if it's working correctly?  Thanks!

```
  // The cookieParser should be above session
  app.use(cookieParser());

  // Request body parsing middleware should be above methodOverride
  app.use(expressValidator());
  app.use(bodyParser.json());
  app.use(bodyParser.urlencoded({
    extended: true
  }));
  app.use(methodOverride());

  // Integrate Express-pouchdb
  app.use('/db', require('express-pouchdb')(require('pouchdb')));
```

**Comment by marten-de-vries** [comment]
I reworded this in #162 to simply warn that middleware may conflict and used bodyParser as an example. Theoretically it **can** conflict, I'm not sure if we test that case though.

**Comment by marten-de-vries** [comment]
Fixed in 29c016b28d93b7dad52d0df5a35bea6ec493da29
